adapters/trustedstack/trustedstack_test.go

```diff
@@ -0,0 +1,31 @@
+package trustedstack
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/prebid/prebid-server/v2/adapters/adapterstest"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderTrustedstack, config.Adapter{
+		Endpoint:         "https://example.trustedstack.com/rtb/prebid",
```

Comment:
static/bidder-info/trustedstack.yaml specifies `https://prebid-adapter.trustedstack.com/rtb/pb/trustedstacks2s` as endpoint 

Requesting to use same address mentioned in bidder-info yaml. This ensures adapter code works with endpoint mentioned in bidder-info yaml.